src/components/top-bar/_top-bar.scss

```diff
@@ -93,6 +93,10 @@
   }
 }
 
+.player .top-bar:has(.left-controls:empty + .right-controls:empty) {
+  visibility: hidden;
```

Comment:
Not necessarily saying this is wrong, but what about display none or changing the dimensions of the element ? Does that work ?

Reply:
it will probably work, but the top-bar is working with visibility property, so it made sense to keep using it.